src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -773,33 +760,20 @@ class ShenandoahScanRemembered: public CHeapObj<mtGC> {
 
 
   // Card index is zero-based relative to first spanned card region.
-  size_t last_valid_index();
-  size_t total_cards();
   size_t card_index_for_addr(HeapWord *p);
   HeapWord *addr_for_card_index(size_t card_index);
   bool is_card_dirty(size_t card_index);
   bool is_write_card_dirty(size_t card_index);
-  void mark_card_as_dirty(size_t card_index);
-  void mark_range_as_dirty(size_t card_index, size_t num_cards);
-  void mark_card_as_clean(size_t card_index);
-  void mark_range_as_clean(size_t card_index, size_t num_cards);
```

Comment:
Are these being removed because the "card_index" arg versions are never used, only the "heapword*" arg versions?

Reply:
Can the "TODO:" at line 748 (old line 761) also be removed, or is there a good reason to keep it for now?

Reply:
Yes, these aren't used. I'm removing that TODO in the next batch after I remove the last of the unused methods.